### PR TITLE
1895 - batch modals

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/presenter/actions/rescanBatchAction.js [hunk 1]
```diff
@@ -18,14 +18,21 @@ export const rescanBatchAction = async ({
...
+      const { scannedBuffer: pages } = await scanner.startScanSession({
+        applicationContext,
+      });
+      const documentSelectedForScan = get(state.documentSelectedForScan);
+      const batches = get(state.batches[documentSelectedForScan]);
+      batches.find(b => b.index === batchIndex).pages = pages;
+      store.set(state.batches[documentSelectedForScan], batches);
+      store.set(state.submitting, false);
+    } catch (err) {
+      if (err.message.indexOf('no images in buffer') === 0) {
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/rescanBatchAction.js]: Could you add a test for this new code? I believe you could also change this statement to `err.message.includes('no images in buffer')` if you'd like to make it a little cleaner.

File: web-client/src/presenter/actions/startScanAction.js [hunk 2]
```diff
@@ -25,25 +25,32 @@ export const startScanAction = async ({
...
+        ...[
+          {
+            index: nextIndex,
+            pages,
+          },
+        ],
+      ]);
+      store.set(state.submitting, false);
+    } catch (err) {
+      if (err.message.indexOf('no images in buffer') === 0) {
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/startScanAction.js]: Same for this, add a test please? Also could switch to `.includes`.

File: web-client/src/views/ScanBatchPreviewer.jsx [hunk 3]
```diff
@@ -1,8 +1,14 @@
-import { ConfirmModal } from '../ustc-ui/Modal/ConfirmModal';
+// import { ConfirmModal } from '../ustc-ui/Modal/ConfirmModal';
```
Comment (rachaelparris) [inline web-client/src/views/ScanBatchPreviewer.jsx]: Remove?

**Review by rachaelparris — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Review by sutt0n — APPROVED** [review]

**Pull request merged**